samples/example-vue-site/core/src/main/java/com/example/site/models/NavModel.java

```diff
@@ -54,7 +54,7 @@ public NavModel(Resource resource) {
     public String getBrand() {
         RenderContext rx = PageMerge.getRenderContext();
         SlingHttpServletRequest request = rx.getRequest();
```

Comment:
The request isn't used anymore. this might be obsolete

Reply:
I removed the unused line. Thanks